src/components/Shared/Audio/CoverImage.tsx

```diff
@@ -0,0 +1,72 @@
+import { Spinner } from '@components/UI/Spinner';
+import { PhotographIcon } from '@heroicons/react/outline';
+import getIPFSLink from '@lib/getIPFSLink';
+import imagekitURL from '@lib/imagekitURL';
+import uploadMediaToIPFS from '@lib/uploadMediaToIPFS';
+import clsx from 'clsx';
+import type { ChangeEvent, FC } from 'react';
+import React, { useState } from 'react';
+import toast from 'react-hot-toast';
+import { ERROR_MESSAGE } from 'src/constants';
+
+type ThumbnailProps = { isNew: boolean; cover: string; setCover: (url: string, mimeType: string) => void };
```

Comment:
**suggestion:** interface bls 🙏🏼 

just name it `Props` ThumbnailProps doesn't match with comp name as well